charts/ingress-nginx/tests/controller-keda_pdb_test.yaml

```diff
@@ -0,0 +1,22 @@
+suite: Controller > Keda > PDB
+templates:
+  - controller-poddisruptionbudget.yaml
+
+tests:
+  - it: should create a PodDisruptionBudget if `controller.kind` is "Deployment" and `controller.keda.enabled` is true
+    set:
+      controller.kind: Deployment
+      controller.keda.enabled: true
+      controller.keda.minReplicas: 2
+      controller.minAvailable: 1
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: PodDisruptionBudget
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-ingress-nginx-controller
+      - equal:
+          path: spec.minAvailable
+          value: 1
```

Comment:
```suggestion
          value: 1
```

Reply:
This suggestions adds the requite line feed at the end of the file.

Reply:
Also this was addressed in the last commit where the file was renamed :)